lib/tasks/sharetribe.rake

```diff
@@ -10,8 +10,7 @@ namespace :sharetribe do
   desc "Generates customized CSS stylesheets in the background"
   task :generate_customization_stylesheets => :environment do
     # If preboot in use, give 2 minutes time to load new code
-    delayed_opts = {priority: 10, :run_at => 2.minutes.from_now }
-    CommunityStylesheetCompiler.compile_all(delayed_opts)
+    CommunityStylesheetCompiler.compile_all(run_at: 2.minutes.from_now)
```

Comment:
Preboot. The 2 minute delayed is there in order to do our best to sync the compilation with the code changes. We don't want to start compiling new stylesheet if it doesn't work with the old code (which will be still running couple of minutes)